packages/web-app/package.json

```diff
@@ -49,7 +49,8 @@
     "sharp": "^0.31.3",
     "tippy.js": "^6.3.7",
     "wagmi": "^0.6.7",
-    "web3.storage": "^4.4.0"
+    "web3.storage": "^4.4.0",
```

Comment:
Can `web3.storage` be deleted?

Reply:
I do think we need to refactor this source code, a lot of outdated libraries